modules/webbrowser/cmake/webbrowser_helpers.cmake

```diff
@@ -160,7 +160,9 @@ function(run_cef_linux_config CEF_TARGET CEF_ROOT)
 endfunction ()
 
 function(set_modules_dependency_on_cef_libraries LIB_DEPENDENT)
-  target_link_libraries(${LIB_DEPENDENT} INTERFACE libcef_lib)
+  if (WIN32 OR OS_LINUX)
+    target_link_libraries(${LIB_DEPENDENT} INTERFACE libcef_lib)
+  endif ()
```

Comment:
This seems like it is just patching another bug;  even on MacOS we ought to link against CEF?

Reply:
I agree, it felt weird taking it out, but I was getting linker errors with it in, and working w/o out it....something to investigate

Reply:
Thought I could be of more assistance, sorry